sources/updater/updog/src/main.rs

```diff
@@ -583,10 +590,10 @@ fn main_inner() -> Result<()> {
 }
 
 fn load_manifest(repository: &tough::Repository) -> Result<Manifest> {
-    let target = "manifest.json";
+    let target = "manifest.json".try_into().unwrap();
```

Comment:
Another error variant is needed here.

Also I might suggest using the type name so a reader can see what type is involved.

```rust
let target = TargetName::new("manifest.json").context(error::InvalidTargetName{ blah blah }?;
```